src/components/List/List.test.tsx

```diff
@@ -12,72 +11,75 @@ describe("List ", () => {
   });
 
   it("renders", () => {
-    const wrapper = shallow(<List items={items} />);
-    expect(wrapper).toMatchSnapshot();
+    render(<List items={items} />);
+    expect(screen.getByRole("list")).toMatchSnapshot();
   });
 
   it("can define items as JSX", () => {
-    const wrapper = shallow(
-      <List items={[<div>test</div>, <span>items</span>]} />
-    );
-    const item = wrapper.find(".p-list__item").first();
-    compareJSX(item.children(), <div>test</div>);
+    render(<List items={[<button>test</button>, <span>items</span>]} />);
+    expect(
+      within(screen.getAllByRole("listitem")[0]).getByRole("button")
+    ).toBeInTheDocument();
   });
 
   it("can define items as objects", () => {
-    const wrapper = shallow(
-      <List items={[{ content: "test", className: "item-class" }]} />
+    render(
+      <List
+        items={[
+          {
+            content: "test",
+            className: "item-class",
+          },
+        ]}
+      />
     );
-    const item = wrapper.find(".p-list__item").first();
-    expect(item.prop("className").includes("item-class")).toBe(true);
+    const item = within(screen.getAllByRole("listitem")[0]).getByText("test");
```

Comment:
Why not directly `screen.getByText("test")`?

Reply:
This test is to check that it constructed a `<li>` (or `listitem` role) from the object that was passed in. If we were to just check that the text was rendered this would pass if it didn't output a list.

Not sure why I used `getAllByRole` here, I'll update it to `screen.getByRole("listitem")`.